src/file/file.service.spec.ts

```diff
@@ -68,6 +76,60 @@ describe('File Logic', () => {
     });
   });
 
+  describe('#createUpload', () => {
+    it('should return a new upload', async () => {
+      const newUpload: IUpload =
+      await FileService.createUpload(upload.uploadID, upload.key, upload.bucket).should.eventually.exist;
+      expect(newUpload).to.exist;
+      expect(newUpload.uploadID).to.be.equal(upload.uploadID);
+      expect(newUpload.bucket).to.be.equal(upload.bucket);
+      expect(newUpload.key).to.be.equal(upload.key);
+    });
+
+    // TODO
+    it.skip('should throw an error when key already exist', async () => {
+      uploadModel.on('index', async (err) => { // <-- Wait for model's indexes to finish
+        console.log('on index');
+        const newUpload1: IUpload =
+        await FileService.createUpload(upload.uploadID, upload.key, upload.bucket)
+        .should.eventually.exist;
+        const newUpload2: IUpload =
+        await FileService.createUpload(upload.uploadID, upload.key, upload.bucket)
+        .should.eventually.be.rejectedWith(KeyAlreadyExistsError);
+        console.log(newUpload1);
+        console.log(newUpload2);
+      });
+
+    });
+  });
+
+  describe('#getUploadById', () => {
+    it('should return a saved upload', async () => {
+      await FileService.createUpload(upload.uploadID, upload.key, upload.bucket);
+      const myUpload = await FileService.getUploadById(upload.uploadID);
```

Comment:
Use more meaningful variable names, in this case it's prefer to just call this variable `upload`.
If it's an upload you expect to not cause an error than give it a name like `badUpload` or `notExistingUpload`, or just keep it simple with `upload` :smiley: .